packages/web/src/components/search/SearchBarV2.jsx

```diff
@@ -0,0 +1,465 @@
+import { createRef, Component } from 'react'
```

Comment:
high level context here is that it was safer to just create a whole new one?

Reply:
Yup, it was difficult to weave the feature flag into the small pieces of logic that needed to change. So decided to make a new one. I think the styling is going to have to change also